nessie/customerRequests.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import utils.constants
+import json
+import re
+from models.customer import Customer
+from utils.exceptions import CustomerValidationError, NessieApiError, AddressValidationError
+
+
+class CustomerRequests:
+
+    def __init__(self, api_key):
+        self.key = api_key
+
+    # Methods Dealing With Customers
+
+    # Get customer for an Account
+    # Returns a Customer object who owns the AccountId
+    def get_customers_by_account_id(self, account_id):
```

Comment:
i'm about to be very nitpicky, but maybe change this to "get_customer_by_account_id" since it's only returning a single customer object